packages/experimental/Shimmer/src/Shimmer.types.ts

```diff
@@ -0,0 +1,135 @@
+import { SvgProps, RectProps, CircleProps, ImageProps } from 'react-native-svg';
+import { ImageURISource } from 'react-native';
+
+export const shimmerName = 'Shimmer';
+/**
+ * Specifies the possible type of the shimmer elements.
+ * Required when building more than one element at a time.
+ */
+export enum ShimmerElementType {
+  /**
+   * Rect element type
+   */
+  rect = 1,
+  /**
+   * Circle element type
+   */
+  circle = 2,
+}
+/**
+ * Represents the shimmer element
+ */
+export interface ShimmerElement {
+  /**
+   * Type of the shimmer element
+   */
+  type?: ShimmerElementType;
+  /**
+   * Specifies the element width, non-op for circle element
+   * @defaultValue '200'
+   */
+  width?: number;
+  /**
+   * Specifies the shimmer view height. For circle element type, height is divided by 2 to get the radius for Svg
+   * @defaultValue '100'
+   */
+  height?: number;
+  /**
+   * Specifies the rect shimmer border radius, non-op for circle element
+   * @defaultValue '0'
+   */
+  borderRadius?: number;
+  /**
+   * x-coordinate
+   * @defaultValue '0'
+   */
+  xPos?: number;
+  /**
+   * y-coordinate
+   * @defaultValue '0'
+   */
+  yPos?: number;
+}
+
+export interface ShimmerTokens {
+  /**
+   * The final value for the animation to reach the end
+   * @defaultValue '30'
+   */
+  toValue?: number;
```

Comment:
I don't think I understand. Is this a duration? or the # of steps in the animation?

Reply:
Yeah it's more like the distance to travel from startValue. For the default case, the animation will animate from 0 to 30 over the course of 7000 milliseconds(duration). Since speed = distance / time, speed ≈ 0.004 (higher the speed, faster it moves). I should probably reword it..

Reply:
That's helpful. I think I'd even appreciate just the default example you specified in documentation.
Like if you put "For the default case, the animation will animate from 0 to 30 over the course of 7000 milliseconds(duration). Since speed = distance / time, speed ≈ 0.004 (higher the speed, faster it moves)" as a comment